### PR TITLE
Adapt integration tests to multi-node setup

### PR DESCRIPTION
**Pull request by andy-slac**

### CODE REVIEW

File: bin/qserv-check-integration.py [hunk 1]
```diff
@@ -68,10 +68,12 @@ def _parse_args():
...
                        help="Test case number")
     mode_option_values = ['mysql', 'qserv', 'all']
-    group.add_argument(
-        "-m", "--mode", dest="mode", choices=mode_option_values,
-        default='all',
-        help="Qserv test modes (direct mysql connection, or via qserv)")
+    group.add_argument("-m", "--mode", dest="mode", choices=mode_option_values,
+                       default='all',
+                       help="Qserv test modes (direct mysql connection, or via qserv)")
+    group.add_argument("-mn", "--multi", action="store_true", dest="multi_node",
```
Comment (andy-slac) [inline bin/qserv-check-integration.py]: `-mn` option may be confusing, usually short option are one-letter, and `-ab` is usually interpreted as `-a -b`. I'd change it to one-letter (e.g. `-u`)

File: bin/qserv-check-integration.py [hunk 2]
```diff
@@ -143,21 +148,26 @@ def _parse_args():
     else:
         args.mode_list = [args.mode]
 
+    if args.multi_node == True and args.mode != 'qserv':
```
Comment (andy-slac) [inline bin/qserv-check-integration.py]: `if args.multi_node == True` can be replaced with just `if args.multi_node`

File: bin/qserv-check-integration.py [hunk 3]
```diff
@@ -143,21 +148,26 @@ def _parse_args():
...
+        _LOG.error("Select --mode=qserv for multi-node test")
+        sys.exit()
```
Comment (andy-slac) [inline bin/qserv-check-integration.py]: `sys.exit()` without argument means success, add non-zero return code to return an error: `sys.exit(2)`

File: python/lsst/qserv/tests/dbLoader.py [hunk 4]
```diff
@@ -31,30 +31,40 @@
...
         self.config = config
         self.dataConfig = data_reader
         self._dbName = db_name
 
+        self._multi_node = multi_node
         self._out_dirname = out_dirname
 
         self.logger = logging.getLogger(__name__)
 
+        if self._multi_node == True:
```
Comment (andy-slac) [inline python/lsst/qserv/tests/dbLoader.py]: `if self._multi_node:`

File: python/lsst/qserv/tests/dbLoader.py [hunk 5]
```diff
@@ -31,30 +31,40 @@
...
+            self.qAdmin = qservAdmin.QservAdmin('localhost:12181')
```
Comment (andy-slac) [inline python/lsst/qserv/tests/dbLoader.py]: Hardcoded port number, qserv loader does `"localhost:" + str(self.config['zookeeper']['port'])`

File: python/lsst/qserv/tests/qservDbLoader.py [hunk 6]
```diff
@@ -75,6 +77,10 @@ def _callLoader(self, table):
 
         loaderCmd += ['--css-remove']
 
+        if self.multi_node == True:
```
Comment (andy-slac) [inline python/lsst/qserv/tests/qservDbLoader.py]: same here

File: python/lsst/qserv/tests/qservDbLoader.py [hunk 7]
```diff
@@ -101,19 +107,21 @@ def _callLoader(self, table):
...
 
         self.logger.info("Drop and create MySQL database for Qserv: %s",
                          self._dbName)
 
-        self.wmgr.dropDb(self._dbName, mustExist=False)
-        self.wmgr.createDb(self._dbName)
+        self.czar_wmgr.dropDb(self._dbName, mustExist=False)
+        self.czar_wmgr.createDb(self._dbName)
+
+        if self.multi_node == True:
```
Comment (andy-slac) [inline python/lsst/qserv/tests/qservDbLoader.py]: and here

File: python/lsst/qserv/tests/qservDbLoader.py [hunk 8]
```diff
@@ -125,5 +133,7 @@ def dropCssDatabase(self):
         self.logger.info("Drop CSS database: %s", self._dbName)
 
     def workerInsertXrootdExportPath(self):
-
-        self.wmgr.xrootdRegisterDb(self._dbName, allowDuplicate=True)
+        if self.multi_node == True:
```
Comment (andy-slac) [inline python/lsst/qserv/tests/qservDbLoader.py]: and here too

File: bin/qserv-check-integration.py (hunk 1 above)
Comment (vaikuntht) [inline bin/qserv-check-integration.py]: Changed option to -M

File: bin/qserv-check-integration.py (hunk 2 above)
Comment (vaikuntht) [inline bin/qserv-check-integration.py]: Changed everywhere this occurs.

File: bin/qserv-check-integration.py (hunk 3 above)
Comment (vaikuntht) [inline bin/qserv-check-integration.py]: Done.

**Pull request merged**